test/testSetup.ts

```diff
@@ -99,11 +103,16 @@ export class TestSetup {
     await expect(uncaughtErrors.length).toBe(0);
   }
 
-  public async setUpTestingEnvironment(): Promise<void> {
+  public async setUpTestingEnvironment(repo?: constants.repoKeywords): Promise<void> {
     utilities.log('');
     utilities.log(`${this.testSuiteSuffixName} - Starting setUpTestingEnvironment()...`);
 
-    this.tempFolderPath = path.join(__dirname, '..', 'e2e-temp');
+    if (repo) {
+      const repoPath = path.join(__dirname, '..', repo);
```

Comment:
Why not use the `this.tempFolderPath` as the root dir for the clone?
```suggestion
      const repoPath = path.join(this.tempFolderPath, repo);
```